src/components/Tabs.tsx

```diff
@@ -51,6 +59,18 @@ export function Tabs(props: Props) {
     });
   };
 
+  useEffect(() => {
+    if (props.tabs.length) {
+      const doesDefaultUrlExist = props.tabs.some(
+        ({ href }) => href === location.pathname
+      );
+
+      if (!doesDefaultUrlExist) {
+        navigate(props.tabs[0].href);
+      }
+    }
+  }, []);
```

Comment:
@beganovich Just quick explanation for this part.

We have a case where the first/default tab can be removed. For example, on the Client show page, the Invoices tab can be removed based on the logic in enabled_modules.

If we keep the default URL, none of the tabs will be selected, and the UI will be rendered incorrectly. Therefore, if the current URL upon mounting this component is not included in the array of tab hrefs, we will simply navigate to the first available tab.